README.old.md

```diff
@@ -0,0 +1,2 @@
+# resty
+An API testing tool that can be run in any browser
```

Comment:
Try to fellow the templet for submission here the [link](https://github.com/codefellows/seattle-javascript-401n19/tree/main/reference/submission-instructions/labs/example) to it

Reply:
Updated the readme.